### PR TITLE
primary color

### PR DESCRIPTION
**Pull request by AgaK-O**
Part of https://tipser.atlassian.net/browse/T3-3082 Jira task concerning Primary Color. 

image

### CODE REVIEW

File: source/includes/_tipser-elements.md [hunk 1]
```diff
@@ -236,4 +236,19 @@ for products on sale.
...
 ```
-The Cart component with the cart icon can be placed anywhere on your website. (It is highly advisable to place it in your navigation element among other icons such as search, home etc.) However, if you want to keep it visible at all times, attached to the right side of the viewport, you can use these styles:
\ No newline at end of file
+The Cart component with the cart icon can be placed anywhere on your website. (It is highly advisable to place it in your navigation element among other icons such as search, home etc.) However, if you want to keep it visible at all times, attached to the right side of the viewport, you can use these styles:
+<br><br><br><br><br>
+
+### Primary color ###
+```css
+.te-primary-fill {
+    background-color: #5F9F9F;
```
Comment (chmurson) [inline source/includes/_tipser-elements.md]: shouldn't we have something like `te-primary-text` then ? I think it is important to have control over a text color used on primary background.

File: source/includes/_tipser-elements.md [hunk 2]
```diff
@@ -236,4 +236,19 @@ for products on sale.
...
+}
+
+.te-primary-border {
+    border-color: #5F9F9F;
+}
+```
+If you'd like to unify our design with your own color-theme, you can use our primary-color theme to change the color of `add-to-cart` buttons in Product and indicator of items number in Cart. These elements on the page will be altered with one small change in your CSS:
```
Comment (piotr-sobczyk) [inline source/includes/_tipser-elements.md]: Are you sure the only thing affected by primary-color setting is add to cart button?

Comment (chmurson) [reply source/includes/_tipser-elements.md]: Yes. Actually all buttons should be affected by this. Also e.g. `Checkout` button on `AddedToCartPopup`.

Comment (chmurson) [reply source/includes/_tipser-elements.md]: My proposition is:

(...) theme to change the color of all buttons such as "Add to cart" in Product, but also indicator of items number in Cart.

Comment (AgaK-O) [reply source/includes/_tipser-elements.md]: @piotr-sobczyk currently the only buttons are `add to cart` but what @chmurson proposed seems legit.

File: source/includes/_tipser-elements.md (hunk 1 above)
Comment (AgaK-O) [reply source/includes/_tipser-elements.md]: `te-primary-text` would mean that we assign the primary color to the text. Makes no sense with the primary color as background-color.
But I'll add a warning to the docs that sometimes the `color: sth` needs to be changed to a contrasting one.

File: source/includes/_tipser-elements.md (hunk 2 above)
Comment (AgaK-O) [reply source/includes/_tipser-elements.md]: @piotr-sobczyk do you have some other elements in mind to be affected by primary color? I might have omitted sth.

**Pull request merged**